### PR TITLE
[WIP] Use SSL if possible

### PR DESCRIPTION
**Pull request by yhoogstrate**
I tried to find all http:// urls in the tool_dependencies.xml in tools-iuc, downloaded them, created md5 sums, replaced http:// for https://, downloaded those as well, matched them with the md5sums, and if there was a match I replaced the XML files.

If you want an overview of the MD5sums:

https://gist.github.com/yhoogstrate/53dd27df439b40b2a44c#file-http_to_https (save as, open with a browser)

@todo: planemo shed_test whether installations still work

### CODE REVIEW

**Comment by erasche** [comment]
[Jenkins Bot] Can one of the admins sign off on testing this PR? (Security reasons, simply comment "ok to test")

File: packages/package_snpeff_3_6/tool_dependencies.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
                     <source_directory>.</source_directory>
                     <destination_directory>$INSTALL_DIR</destination_directory>
                 </action>
-                <action type="shell_command">sed -i.orig -e 's;http://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
+                <action type="shell_command">sed -i.orig -e 's;https://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
```
Comment (nsoranzo) [inline packages/package_snpeff_3_6/tool_dependencies.xml]: This is wrong, must be reverted.

File: packages/package_snpeff_4_0/tool_dependencies.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
                     <source_directory>.</source_directory>
                     <destination_directory>$INSTALL_DIR</destination_directory>
                 </action>
-                <action type="shell_command">sed -i.orig -e 's;http://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
+                <action type="shell_command">sed -i.orig -e 's;https://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
```
Comment (nsoranzo) [inline packages/package_snpeff_4_0/tool_dependencies.xml]: This is wrong, must be reverted.

File: packages/package_snpeff_4_1/tool_dependencies.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
                     <source_directory>.</source_directory>
                     <destination_directory>$INSTALL_DIR</destination_directory>
                 </action>
-                <action type="shell_command">sed -i.orig -e 's;http://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
+                <action type="shell_command">sed -i.orig -e 's;https://downloads.sourceforge.net/project/snpeff/databases;https://snpeff-data.galaxyproject.org/databases;g' $INSTALL_DIR/snpEff.config</action>
```
Comment (nsoranzo) [inline packages/package_snpeff_4_1/tool_dependencies.xml]: This is wrong, must be reverted.

File: packages/package_snpeff_3_6/tool_dependencies.xml (hunk 1 above)
Comment (yhoogstrate) [inline packages/package_snpeff_3_6/tool_dependencies.xml]: Thanks!

File: packages/package_snpeff_4_0/tool_dependencies.xml (hunk 2 above)
Comment (yhoogstrate) [inline packages/package_snpeff_4_0/tool_dependencies.xml]: Thanks!

File: packages/package_snpeff_4_1/tool_dependencies.xml (hunk 3 above)
Comment (yhoogstrate) [inline packages/package_snpeff_4_1/tool_dependencies.xml]: Thanks!

File: packages/package_monocle_1_0_0/tool_dependencies.xml [hunk 4]
```diff
@@ -10,35 +10,35 @@
...
-                    <package>http://bioconductor.org/packages/3.0/bioc/src/contrib/Biobase_2.26.0.tar.gz</package>
-                    <package>http://cran.fhcrc.org/src/contrib/Rcpp_0.11.4.tar.gz</package>
-                    <package>http://cran.fhcrc.org/src/contrib/plyr_1.8.1.tar.gz</package>
+                    <package>https://cran.fhcrc.org/src/contrib/VGAM_0.9-6.tar.gz</package>
+                    <package>https://bioconductor.org/packages/3.0/data/experiment/src/contrib/HSMMSingleCell_1.0.0.tar.gz</package>
+                    <package>https://bioconductor.org/packages/3.0/bioc/src/contrib/BiocGenerics_0.12.1.tar.gz</package>
+                    <package>https://bioconductor.org/packages/3.0/bioc/src/contrib/Biobase_2.26.0.tar.gz</package>
+                    <package>https://cran.fhcrc.org/src/contrib/Rcpp_0.11.4.tar.gz</package>
+                    <package>https://cran.fhcrc.org/src/contrib/plyr_1.8.1.tar.gz</package>
                     <package>http://cran.fhcrc.org/src/contrib/digest_0.6.8.tar.gz</package>
```
Comment (yhoogstrate) [inline packages/package_monocle_1_0_0/tool_dependencies.xml]: Many broken urls:

--2015-07-13 19:46:45--  http://cran.fhcrc.org/src/contrib/VGAM_0.9-6.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:46 ERROR 404: Not Found.

--2015-07-13 19:46:46--  http://cran.fhcrc.org/src/contrib/Rcpp_0.11.4.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:46 ERROR 404: Not Found.

--2015-07-13 19:46:46--  http://cran.fhcrc.org/src/contrib/plyr_1.8.1.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:47 ERROR 404: Not Found.

--2015-07-13 19:46:49--  http://cran.fhcrc.org/src/contrib/stringr_0.6.2.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:49 ERROR 404: Not Found.

--2015-07-13 19:46:53--  http://cran.fhcrc.org/src/contrib/colorspace_1.2-4.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:53 ERROR 404: Not Found.

--2015-07-13 19:46:55--  http://cran.fhcrc.org/src/contrib/scales_0.2.4.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:56 ERROR 404: Not Found.

--2015-07-13 19:46:58--  http://cran.fhcrc.org/src/contrib/MASS_7.3-39.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:58 ERROR 404: Not Found.

--2015-07-13 19:46:58--  http://cran.fhcrc.org/src/contrib/ggplot2_1.0.0.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:59 ERROR 404: Not Found.

--2015-07-13 19:46:59--  http://cran.fhcrc.org/src/contrib/lattice_0.20-30.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:46:59 ERROR 404: Not Found.

--2015-07-13 19:46:59--  http://cran.fhcrc.org/src/contrib/Matrix_1.1-5.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:47:00 ERROR 404: Not Found.

--2015-07-13 19:47:00--  http://cran.fhcrc.org/src/contrib/igraph_0.7.1.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:47:00 ERROR 404: Not Found.

--2015-07-13 19:47:00--  http://cran.fhcrc.org/src/contrib/cluster_2.0.1.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:47:00 ERROR 404: Not Found.

--2015-07-13 19:47:04--  http://cran.fhcrc.org/src/contrib/matrixStats_0.14.0.tar.gz
Resolving cran.fhcrc.org (cran.fhcrc.org)... 140.107.3.21
Connecting to cran.fhcrc.org (cran.fhcrc.org)|140.107.3.21|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:47:05 ERROR 404: Not Found.

--2015-07-13 19:47:05--  http://bioconductor.org/packages/3.0/bioc/src/contrib/limma_3.22.4.tar.gz
Resolving bioconductor.org (bioconductor.org)... 54.230.12.6, 54.192.14.12, 54.230.15.64, ...
Connecting to bioconductor.org (bioconductor.org)|54.230.12.6|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2015-07-13 19:47:05 ERROR 404: Not Found.

**Comment by erasche** [comment]
Now part of #213

**Pull request closed without merging**